### DISCUSSION TITLE
Mathematical equations

### DISCUSSION THREAD

**Discussion by DB11051998** [mindee/doctr#1161] [General]
Does this support mathematical equations in prediction?

**Comment by Zephkek** [comment]
Smae question

**Comment by felixdittrich92** [comment]
Hi @DB11051998 @Zephkek :wave:,

For some basic equations it will work but nothing (scientific) more complex.

Take a look here: https://github.com/lukas-blecher/LaTeX-OCR

Best regards,
Felix
